specs/BotBuilderOpenSourcePlans.md

```diff
@@ -0,0 +1,85 @@
+3. [Java SDK v4](https://botbuilder.myget.org/gallery/botbuilder-v4-java-daily)
+
+Following the Python Best practices as defined by TensorFlow, the daily builds for the Python SDK will be here:
+[https://pypi.org/project/botbuilder-nightly/](https://pypi.org/project/botbuilder-nightly/)
+
+# External Pull Requests
+
+We love external pull requests. For submitted PR's we require:
+1. Follow the contribution guidelines
+2. Signed the relevant CLA
+3. Follow coding standards for relevant platform
+4. Unit Tests all pass. 
+
+The BotBuilder team is following the [.Net Core FX/CLR contribution guidelines](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/contributing.md). 
+
+Each PR will be assigned to a BotBuilder team member for review and reviewed against the relevant guideliens. 
+
+# Stand-up
+The ASP.Net team does a [weekly public standup](https://live.asp.net/). At this time, the BotBuilder team doesn't think this is needed. Any voice/video discussions that need to happen can be scheduled, but most discussion should take place via GitHub. 
+
+# External Speaking Engagments
+Requests for presentations at local groups should be submitted via GitHub Issues. Each request will be seperatly considered, and community members may choose to engage. As the community grows we hope this becomes an active part of our broad community.
+
+# Repo Layout and Usage
+
+In Github, we are going with 5 repos. These break down as:
+1. BotBuilder. This is the "root" repo were overviews, docs, and links to other repos are found. By design, this is the "landing page" for Bot developers. There should be no source-code found in this repo. 
+2. Botbuilder-dotnet. This is the C# Repo. Contained here is the V3 and V4 C# Code, along with tests, relevant samples, and issues. Links to the daily preview build, the public build, and other platform relevant materials are found here.
+3. Botbuilder-js.  This is the JavaScript Repo. Contained here is the V3 and V4 Javascript Code, along with tests, relevant samples, and issues. Links to the daily preview build for Node, the public build for Node, and other platform relevant materials are found here. 
+4. BotBuilder-Java. This is the Java Repo. Contained here is the V4 Java Code, along with tests, relevant samples, and issues. Links to the daily preview build, the public build, and other platform relevant materials are found here.
+5. BotBuilder-Python. This is the Python Repo. Contained here is the V4 Python Code, along with tests, relevant samples, and issues. Links to the daily preview build, the public build, and other platform relevant materials are found here.
+
+By design, most Issues and discussions should happen on the platform-agnostic "Botbuilder" repo. Issues that are incorrectly filed on a platform specific repo will be closed and re-opened on the correct repo. Likewise, platform specific issues will be moved as needed. 
+
```

Comment:
@cleemullins  we should document that: Master is the latest and greatest, release branches are snapped monthly (or per release).